packages/manager/modules/pci/src/projects/project/private-registry/private-registry.controller.js

```diff
@@ -0,0 +1,94 @@
+import get from 'lodash/get';
+import { PRIVATE_REGISTRY_STATUS, PRIVATE_REGISTRY_STATUS_MAP, GUIDELINK } from './private-registry.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $state,
+    $stateParams,
+    CucCloudMessage,
+    privateRegistryService,
+  ) {
+    this.$state = $state;
+    this.$stateParams = $stateParams;
+    this.projectId = $stateParams.projectId;
+    this.CucCloudMessage = CucCloudMessage;
+    this.privateRegistryService = privateRegistryService;
+    this.guideLink = GUIDELINK;
+    this.PRIVATE_REGISTRY_STATUS = PRIVATE_REGISTRY_STATUS;
+    this.PRIVATE_REGISTRY_STATUS_MAP = PRIVATE_REGISTRY_STATUS_MAP;
+    this.registryList = [];
+  }
+
+  $onInit() {
+    this.loadMessages();
+    this.getRegistryList(false);
```

Comment:
You could use `registries` as it is resolved when the component is initialized (https://github.com/ovh-ux/manager/pull/698/files#diff-5b6748c3dbe9a118fca173c356241742R28).
We should avoid to call the API in `$onInit` and use uirouter `resolves`, as if one `resolve` fail, the transition will throw an error and we redirect the user to a generic error page (https://github.com/ovh-ux/manager/blob/master/packages/manager/apps/public-cloud/src/index.js#L75-L84)